core-bundle/src/Resources/contao/templates/modules/mod_breadcrumb.html5

```diff
@@ -1,15 +1,24 @@
-<?php $this->extend('block_unsearchable'); ?>
 
-<?php $this->block('content'); ?>
+<!-- indexer::stop -->
+<nav class="<?= $this->class ?> block"<?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?> aria-label="Breadcrumb" itemscope itemtype="http://schema.org/BreadcrumbList">
 
-  <ul itemprop="breadcrumb" itemscope itemtype="http://schema.org/BreadcrumbList">
```

Comment:
Did you remove `itemprop="breadcrumb"` on purpose?

Reply:
Of course not, fixed :)